README.md

```diff
@@ -19,7 +19,42 @@ using AbstractDifferentiation
 
 ## `AbstractDifferentiation` backends
 
-To use `AbstractDifferentiation`, first construct a backend instance `ab::AD.AbstractBackend` using your favorite differentiation package in Julia that supports `AbstractDifferentiation`. For higher order derivatives, you can build higher order backends using `AD.HigherOrderBackend`. For instance, let `ab_f` be a forward-mode automatic differentiation backend and let `ab_r` be a reverse-mode automatic differentiation backend. To construct a higher order backend for doing forward-over-reverse-mode automatic differentiation, use `AD.HigherOrderBackend((ab_f, ab_r))`. To construct a higher order backend for doing reverse-over-forward-mode automatic differentiation, use `AD.HigherOrderBackend((ab_r, ab_f))`.
+To use `AbstractDifferentiation`, first construct a backend instance `ab::AD.AbstractBackend` using your favorite differentiation package in Julia that supports `AbstractDifferentiation`.
+The following backends are made available by `AbstractDifferentiation` if their corresponding package is loaded (thanks to [Requires.jl](https://github.com/JuliaPackaging/Requires.jl)):
+
+- `AD.ForwardDiffBackend()` for [ForwardDiff.jl](https://github.com/JuliaDiff/ForwardDiff.jl)
+- `AD.FiniteDifferencesBackend()` for [FiniteDifferences.jl](https://github.com/JuliaDiff/FiniteDifferences.jl)
+- `AD.ReverseDiffBackend()` for [ReverseDiff.jl](https://github.com/JuliaDiff/ReverseDiff.jl)
+- `AD.ReverseRuleConfigBackend(ruleconfig)` for any [ChainRules.jl](https://github.com/JuliaDiff/ChainRules.jl)-compatible reverse mode backend
```

Comment:
This is always available and not dependent on Requires. `ZygoteBackend` is just a special variant of it, with `ZygoteRuleConfig` as `ruleconfig` (as you can see below). For other CR-compatible AD backends one would use their respective rule configs.

Reply:
Of course it is not dependent on Requires being loaded. What I meant is that AbstractDifferentation uses Requires to provide this in a transparent manner (see [here](https://github.com/JuliaDiff/AbstractDifferentiation.jl/blob/12a0e27d0f54e6300b725dfb9dafc21ef507b724/src/AbstractDifferentiation.jl#L648)).

As an AD novice, I'm not yet sure what a `RuleConfig` is, so I guess it wouldn't hurt to add an example of that too?

Reply:
No, my point is that this backend does not use Requires. It is always available, even if you don't load any AD package.

`RuleConfig` is a CR thing: https://juliadiff.org/ChainRulesCore.jl/stable/rule_author/superpowers/ruleconfig.html
For AD packages that are compatible with CR an AD-specific `RuleConfig` should be defined: https://juliadiff.org/ChainRulesCore.jl/stable/rule_author/superpowers/ruleconfig.html#Writing-rules-that-are-only-for-your-own-AD E.g., Zygote defines a `ZygoteRuleConfig` `RuleConfig` that allows to call (back) into the Zygote AD pipeline.

You already have an example here, since `ZygoteBackend() == ReverseRuleConfigBackend(Zygote.ZygoteRuleConfig())` :smile:

Reply:
Ah, right, I hadn't understood that the first part of the sentence referred to `ReverseRuleConfigBackend` since the whole list was highlighted, my bad :blush: